src/helpers/MetricDatasourceHelper.ts

```diff
@@ -75,7 +75,7 @@ export class MetricDatasourceHelper {
     const loadMetadata =
       typeof ds.languageProvider.retrieveMetricsMetadata === 'function'
         ? () => Promise.resolve(ds.languageProvider.retrieveMetricsMetadata())
-        : () => (ds.languageProvider.loadMetricsMetadata() as unknown as Promise<void>).then(queryMetadata); // eslint-disable-line sonarjs/deprecation
+        : () => (ds.languageProvider.loadMetricsMetadata() as unknown as Promise<void>).then(() => queryMetadata()); // eslint-disable-line sonarjs/deprecation
```

Comment:
why would this be different?

Reply:
@grafakus We cannot because `loadMetricsMetadata` is marked for deprecation. When metrics-drilldown stop supporting old version of grafana you can delete and only use new methods. 

@shelldandy Not sure but I think `queryMetadata` was expecting an optional limit parameter. Typescript didn't like the previous version I made this.